app/services/create_account.rb

```diff
@@ -41,6 +42,10 @@ def create_redis_namespace
     CreateRedisNamespaceJob.perform_later(account)
   end
 
+  def create_default_admin_set
+    CreateDefaultAdminSetJob.perform_later(account)
```

Comment:
This has to happen after the Fcrepo & Solr collection is created. Presently we don't have any way of making that assurance.  Perhaps we need an "InitializeClientJob".